rero_ils/modules/collections/views.py

```diff
@@ -60,7 +60,7 @@ def collection_view_method(pid, record, template=None, **kwargs):
         item['document'] = Document.get_record_by_pid(
             item.replace_refs().get('document').get('pid'))
     # Get libraries names
-    for library in rec.get('libraries'):
+    for library in rec.get('libraries', []):
```

Comment:
Do we have a libraries key in rec if we do not have library: see line: 67.

Reply:
Done